### PR TITLE
Change require.js to webpack for tests

### PR DESCRIPTION
**Pull request by lboyette-okta**
bacon: test
resolves: OKTA-90172

@austinstanhope-okta 
@rchild-okta

### CODE REVIEW

File: test/main.js [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
-    jquery: '../node_modules/jquery/dist/jquery',
-    lodash: '../node_modules/lodash/lodash',
-    OktaAuth: '../dist/browser/OktaAuthRequireJquery',
-    q: '../node_modules/q/q',
-    sinon: '../node_modules/sinon/lib/sinon',
-    text: '../node_modules/requirejs-text/text'
-  }
-});
+function requireAll(requireContext) {
+	requireContext.keys().map(requireContext);
```
Comment (rchild-okta) [inline test/main.js]: nit: spacing

**Comment by rchild-okta** [comment]
A nit about spacing, and it looks like the tests are failing for the same grunt error as before - is this commit rebased off a version of master that has cache:false?

After resolving, 🚀

File: test/main.js (hunk 1 above)
Comment (lboyette-okta) [inline test/main.js]: will fix

**Comment by austinstanhope-okta** [comment]
Looks ok to me.

**Pull request merged**